src/containers/monitoring/NetDataSettingsForm.tsx

```diff
@@ -37,10 +37,10 @@ export default class NetDataSettingsForm extends Component<{
         <br />
         <h4>Email (SMTP)</h4>
 
-        <Row>
-          <Col span={20} offset={2}>
+        <Row type="flex" justify="center">
```

Comment:
This distorts the view in desktop mode:

Reply:
![Screenshot from 2019-03-23 10-01-31](https://user-images.githubusercontent.com/10582082/54869216-b9b7d380-4d52-11e9-9552-4e6a8ac8d4c2.png)